BraveWallet/Crypto/Stores/AccountActivityStore.swift

```diff
@@ -124,7 +127,12 @@ class AccountActivityStore: ObservableObject {
     allTokens: [BraveWallet.BlockchainToken],
     assetRatios: [String: Double]
   ) async -> [TransactionSummary] {
-    await txService.allTransactionInfo(.eth, from: account.address)
+    let transactions = await txService.allTransactionInfo(account.coin, from: account.address)
+    var solEstimatedTxFees: [String: UInt64] = [:]
+    if network.coin == .sol {
```

Comment:
i think we should use `account.coin` here as well, since all transactions were fetched using `account.coin`.
or maybe inside method `estimatedTxFees` we can check if transaction's coin type is .sol

Reply:
Fixed in [700a537](https://github.com/brave/brave-ios/pull/5737/commits/700a537f6ca35a579f1e16f6cf526f262d9f1d03)